cmd/go-clang-generate/function.go

```diff
@@ -135,3 +139,50 @@ func generateFunctionEqual(f *Function) string {
 
 	return b.String()
 }
+
+var templateGenerateStructMemberGetter = template.Must(template.New("go-clang-generate-function-getter").Parse(`{{$.Comment}}
+func ({{$.Receiver}} {{$.ReceiverType}}) {{$.Name}}() {{if $.IsReturnTypePointer}}*{{end}}{{if $.IsReturnTypeArray}}[]{{end}}{{if $.ReturnPrimitiveType}}{{$.ReturnPrimitiveType}}{{else}}{{$.ReturnType}}{{end}} {
+	return {{if $.IsReturnTypePointer}}&{{end}}{{if $.IsReturnTypeArray}}[]{{end}}{{if $.ReturnPrimitiveType}}{{$.ReturnPrimitiveType}}{{else}}{{$.ReturnType}}{{end}}{{if $.ReturnPrimitiveType}}({{$.Receiver}}.c.{{$.Member}}){{else}}{{"{"}}{{$.Receiver}}.c.{{$.Member}}{{"}"}}{{end}}
+}
+`))
+
+func generateFunctionStructMemberGetter(f *Function) string {
+	var b bytes.Buffer
+	if err := templateGenerateStructMemberGetter.Execute(&b, f); err != nil {
+		panic(err)
+	}
+
+	return b.String()
+}
+
+type FunctionSliceReturn struct {
+	Function
+
+	ElementType          string
```

Comment:
We have a mix here. "ElementType" in your code and "PrimitiveType" in mine. How about doing the renaming of "Conversion" to "Type" and then rename the member "GoType" to "Name" and add a "CName" to it which then gets both roles, "ElementTyper" and "PrimitiveType"?